frontend/src/Hints.js

```diff
@@ -58,6 +58,10 @@ const Hints = ({hints, numHints, requestingSolution, solution}) =>
           />
       }
     </div>
+    {
+      numHints !== hints.length &&
+      <span className="small">Shown {numHints} of {hints.length} hints.</span>
```

Comment:
```suggestion
      numHints < hints.length &&
      <span className="small">Shown {numHints} of {hints.length} hints</span>
```

I moved a step and it said "Shown 10 of 5"